utils.js

```diff
@@ -89,6 +95,9 @@ function sendPing(key, window, details) {
   payload[key] = 1;
   Object.assign(payload, details);
 
+  // Send metrics to shield telemetry
+  TCStudy.report(payload);
```

Comment:
yeah, I wasn't certain about that. I thought on the outline doc we wanted to send to both?